app/helpers/and.js

```diff
@@ -0,0 +1,4 @@
+import Ember from 'ember';
+import { andHelper } from 'ember-truth-helpers/helpers/and';
+
+export default Ember.Helper.helper(andHelper);
```

Comment:
I will not promise that I will have time for restructuring tests for a few days ahead, so please take your time to do this.

@jmurphyau What is the plan for supported Ember versions? Will this project mimic what others do following Ember major.minor version? E.g 1.0.0 is compatible from Ember < 1.13, then release 1.13, and then 2.0 to remove deprecated API usage.